### PR TITLE
Update Luigi tutorials

### PR DESCRIPTION
**Pull request by alexandra-simeonova**
NOTE: This PR is not ready for merge/review. It requires some more changes from the development team. It's easier for the team to see changes in the form of PR in the GitHub environment, which is the reason for opening this draft PR. When it is ready, the draft status will be removed.

Update Luigi tutorial sample: 

- Remove outdated dependencies
- Update to React 18
- Add new example for the Luigi tutorials at https://developers.sap.com/group.luigi-app.html

### CODE REVIEW

File: tutorials/luigi-app-basic-setup/luigi-app-basic-setup.md [hunk 1]
```diff
@@ -33,86 +33,49 @@ In this step, you will create a React skeleton project which will be used to cre
...
     ​
-7. Download the Luigi React configuration:
+7. Download the Luigi React simple app files:
 
     ```Shell
-     curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/public/luigi-config.js > src/luigi-config/luigi-config.es6.js
+    curl https://raw.githubusercontent.com/SAP/luigi/main/core/examples/luigi-example-react/src/index.js > src/index.js
+    curl https://raw.githubusercontent.com/SAP/luigi/main/core/examples/luigi-example-react/src/views/home.js > src/views/home.js
+    curl https://raw.githubusercontent.com/SAP/luigi/main/core/examples/luigi-example-react/src/views/sample1.js > src/views/sample1.js
+    curl https://raw.githubusercontent.com/SAP/luigi/main/core/examples/luigi-example-react/src/views/sample1.js > src/views/sample2.js
```
Comment (ndricimrr) [inline tutorials/luigi-app-basic-setup/luigi-app-basic-setup.md:71]:
```suggestion
    curl https://raw.githubusercontent.com/SAP/luigi/main/core/examples/luigi-example-react/src/views/sample1.js > src/views/sample2.js
    curl https://raw.githubusercontent.com/SAP/luigi/main/core/examples/luigi-example-react/public/index.html > public/index.html
    curl https://github.com/SAP/luigi/blob/main/core/examples/luigi-example-react/public/sampleapp.html > public/sampleapp.html
```

File: tutorials/luigi-app-basic-setup/luigi-app-basic-setup.md [hunk 2]
```diff
@@ -33,86 +33,51 @@ In this step, you will create a React skeleton project which will be used to cre
...
+     curl https://raw.githubusercontent.com/SAP/luigi/main/core/examples/luigi-example-react/webpack.config.js > webpack.config.js
     ```
 
-6. Create a new folder:
+6. Next we need to create the project structure for our simple React app. Create the following folders:
 
     ```Shell
-    mkdir -p src/luigi-config
     mkdir src/views
+    mkdir src/assets
```
Comment (ndricimrr) [inline tutorials/luigi-app-basic-setup/luigi-app-basic-setup.md:62]:
```suggestion
    mkdir -p src/views
    mkdir -p src/assets
```

File: tutorials/luigi-app-basic-setup/luigi-app-basic-setup.md [hunk 3]
```diff
@@ -33,86 +33,51 @@ In this step, you will create a React skeleton project which will be used to cre
...
     ```
 
-6. Create a new folder:
+6. Next we need to create the project structure for our simple React app. Create the following folders:
 
     ```Shell
-    mkdir -p src/luigi-config
-    mkdir src/views
+    mkdir -p src/views
+    mkdir -p src/assets
```
Comment (ndricimrr) [inline tutorials/luigi-app-basic-setup/luigi-app-basic-setup.md:62]:
```suggestion
    mkdir -p src/assets
    mkdir -p public
```

File: tutorials/luigi-app-react/luigi-app-react.md [hunk 4]
```diff
@@ -191,24 +189,48 @@ In this step, you will create another HTML file which will serve as a template f
 
 In this step, we configure `webpack` and adjust dependencies in order to make it easier to develop and build the app.
 
-1. Go to `react-core-mf/config/webpack.config.js`
+1. Go to `react-core-mf/webpack.config.js`
 
-2. Find the following line from the file and [comment it out](https://www.w3schools.com/js/js_comments.asp) by surrounding it with `/*` or `//` tags:
+2. **Note** the following line. By default our `webpack-dev-server` will redirect all requests to `index.html` which is our app's main entry point where the Luigi Core library is injected.
```
Comment (MichaelCzcz) [inline tutorials/luigi-app-react/luigi-app-react.md:194]: Don't use the label "Note" and use the correct formatting for notes as described in the Tutorials style guide.
Also, there is a comma missing feore "which".

File: tutorials/luigi-app-react/luigi-app-react.md [hunk 5]
```diff
@@ -191,24 +189,48 @@ In this step, you will create another HTML file which will serve as a template f
...
+   However since the microfrontends are also built in the same environemnt, we will need to host the root React microfrontend app inside another file, which we have named `sampleapp.html`. A similar approach can be chosen with any other frontend framework:
```
Comment (MichaelCzcz) [inline tutorials/luigi-app-react/luigi-app-react.md:195]: Use You and not the royal We.(unless, in rare cases, you are referring to SAP).

File: tutorials/luigi-app-basic-setup/luigi-app-basic-setup.md [hunk 6]
```diff
@@ -147,7 +114,7 @@ In this step, you will create a skeleton UI5 project for your UI5 micro-frontend
     yo easy-ui5
     ```
 
-5. Choose `generator-ui5-project`. Use the following settings:
+5. Choose `app` and use the settings shown in the screenshot below. For the questions "Which framework do you want to use?" and "Who is the author of this application?", choose the default option and press Enter. 
 
     ![UI5 Terminal](ui5-yo.png)
```
Comment (MichaelCzcz) [inline tutorials/luigi-app-basic-setup/luigi-app-basic-setup.md:119]: Add a border to this image and use parser v2 syntax!

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please address comments and resolve conflicts. 
Always pull from Github before pushing to prevent future conflicts!

File: tutorials/luigi-app-react/luigi-app-react.md [hunk 7]
```diff
@@ -552,18 +508,20 @@ In this step, you will add a navigation node in Luigi for the "Products" micro-f
...
     //AROUND LINE 35 PASTE THIS LINE:
-    <BrowserRouter basename={`sampleapp.html#`}>
+    <Router basename="microfrontend">
+      <Routes>
         <Route path="/home" render={(props) => <Home {...props} localeDict={dict[this.state.currentLocale]} currentLocale={this.state.currentLocale} />} />
-      //THIS LINE HAS TO BE ADDED:
-        <Route path="/products" render={(props) => <Products {...props} localeDict={dict[this.state.currentLocale]} />} />
-    </BrowserRouter>
+        //THIS LINE HAS TO BE ADDED:
+        <Route path="/products" element={<Product localeDict={dict[currentLocale]} />} />
```
Comment (alexandra-simeonova) [inline tutorials/luigi-app-react/luigi-app-react.md:515]:
```suggestion
        <Route path="/products" element={<Products localeDict={dict[currentLocale]} />} />
```

File: tutorials/luigi-app-localization-ui5/luigi-app-localization-ui5.md [hunk 8]
```diff
@@ -58,14 +58,14 @@ In this step, you will add a function to get the current language from Luigi Cli
 
 In this step, you will create files with the text that is to be changed within the UI5 micro-frontend.
 
-1. Find the folder ​`i18n`​ under the ​`uimodule/webapp`​.  Inside it, create a file called `i18n_de_DE.properties` with the following content:
+1. Find the folder ​`i18n`​ under the ​`webapp`​.  Inside it, create a file called `i18n_de.properties` with the following content:
 
     ```json
     ModalText = Produkt Details
     Quantity = Anzahl
```
Comment (ndricimrr) [inline tutorials/luigi-app-localization-ui5/luigi-app-localization-ui5.md:65]:
```suggestion
    ModalText = Produktdetails
    Quantity = Anzahl
    appTitle = ui5
    appDescription = Deutsch
```

File: tutorials/luigi-app-localization-ui5/luigi-app-localization-ui5.md [hunk 9]
```diff
@@ -58,14 +58,14 @@ In this step, you will add a function to get the current language from Luigi Cli
...
     ```
 
-2. Create another file called `i18n_en_US.properties` with the following content:
+2. Create another file called `i18n_en.properties` with the following content:
 
     ```json
     ModalText = Product Details
```
Comment (ndricimrr) [inline tutorials/luigi-app-localization-ui5/luigi-app-localization-ui5.md:71]:
```suggestion
    ModalText = Product Details
    appTitle = ui5
    appDescription = English
```

File: tutorials/luigi-app-localization-ui5/luigi-app-localization-ui5.md [hunk 10]
```diff
@@ -101,7 +101,8 @@ In this step, you will create files with the text that is to be changed within t
 
 This step involves the standard process in UI5 for providing translation.
 
-1. Edit the ​`ui5-mf/uimodule/webapp/view/Order.view.xml` ​file by marking the translated target text. Replace line 17 with:
+// This file does not exist ?
+1. Edit the ​`ui5-mf/webapp/view/Order.view.xml` ​file by marking the translated target text. Replace line 17 with:
```
Comment (ndricimrr) [inline tutorials/luigi-app-localization-ui5/luigi-app-localization-ui5.md:105]:
```suggestion
1. Edit the ​`ui5-mf/webapp/view/Main.view.xml` ​file by marking the translated target text. Replace `<ObjectAttribute>` tag with:
```

**Review by ndricimrr — COMMENTED** [review]
Some last comments

**Comment by alexandra-simeonova** [comment]
Closing due to branch created wrongly before pulling latest state

**Pull request closed without merging**